vignettes/plots.Rmd

```diff
@@ -19,16 +19,14 @@ knitr::opts_chunk$set(
 
 library(tidyverse)
 library(cmapplot)
+unapply_cmap_default_aes()
```

Comment:
I don't understand why `unapply` is needed here.

Reply:
This was necessary for testing purposes. Without it, repeat knitting of the vignette within an R session causes all the early plots to be drawn with cmap aesthetics. Once the vignette is done being developed, it is no longer useful. However, because this function simply puts back the defaults that were just extracted in the previous line of code, it strikes me as pretty harmless.

Reply:
Ok, just wanted to make sure it wasn't doing something critical that I was unaware of.